app/controllers/course/experience_points_records_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Course::ExperiencePointsRecordsController < Course::ComponentController
```

Comment:
Some of the controllers don't have the frozen string literal line. In what cases should we omit it?